### PR TITLE
Add refresh token tests

### PR DESCRIPTION
**Pull request by Horneth**

### CODE REVIEW

**Comment by geoffjentry** [comment]
👍 

[Approved with PullApprove](https://pullapprove.com/broadinstitute/centaur/pull-request/107/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=broadinstitute/centaur)

File: src/main/resources/standardTestCases/refresh_token_no_auth_bucket.test [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+files {
+  wdl: hello/hello.wdl
+  inputs: hello/hello.inputs
+  options: refresh_token_no_auth_bucket/refresh_token_no_auth_bucket.options
+}
+
+metadata {
+  workflowName: hello
+  status: Failed
```
Comment (cjllanwarne) [inline src/main/resources/standardTestCases/refresh_token_no_auth_bucket.test:13]: Is it possible/excessive to put a failure message in too?

File: src/main/resources/standardTestCases/refresh_token_failure.test [hunk 2]
```diff
@@ -0,0 +1,14 @@
+name: refresh_token_failure
+testFormat: workflowfailure
+backends: [Jes-Refresh]
+
+files {
+  wdl: hello/hello.wdl
+  inputs: hello/hello.inputs
```
Comment (cjllanwarne) [inline src/main/resources/standardTestCases/refresh_token_failure.test:7]: Is there any way to somehow test that input reading is included in the refresh token auth?

Comment (cjllanwarne) [reply src/main/resources/standardTestCases/refresh_token_failure.test:7]: Oh, and that call-caching uses the right auth!

**Review by cjllanwarne — APPROVED** [review]
Looks good. I'm assuming we already have a "Success" version of this test.
Also included: some ideas for "future work" in case it's worth making tickets for them

**Comment by cjllanwarne** [comment]
👍 

[Approved with PullApprove](https://pullapprove.com/broadinstitute/centaur/pull-request/107/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=broadinstitute/centaur)

File: src/main/resources/standardTestCases/refresh_token_no_auth_bucket.test (hunk 1 above)
Comment (Horneth) [reply src/main/resources/standardTestCases/refresh_token_no_auth_bucket.test:13]: Done

**Pull request merged**